conf/modules.config

```diff
@@ -385,16 +385,6 @@ process {
         ext.args = '--coord presto --rc tail --1f CONSCOUNT PRCONS --2f CONSCOUNT PRCONS --failed'
         ext.args2 = '-f ID BARCODE SEQCOUNT PRIMER PRCOUNT PRCONS PRFREQ CONSCOUNT LENGTH OVERLAP ERROR PVALUE'
     }
-    
```

Comment:
As for assemblepairs_sequential, it's a solution I'd already discussed with the team and I don't remember exactly why, but they didn't want to use the reference, I can ask them again if necessary.
It turns out that assemblepairs_join really inflates our numbers of productive sequences at the end of the pipeline, so this option is necessary to recover more than 40% of sequences that wouldn't merge without it.